### ISSUE TITLE
Incompatable with Mekanism

### ISSUE THREAD

**Issue by Acat1234** [Yesssssman/epicfightmod#1426]
### Have you checked if a similar issue is already reported by someone else?

- [X] I checked there are no similar issues have been reported.

### Have you read the support policy?

- [X] I read it and I accept the policy.

### Are you using the latest Epic Fight and recommended Forge version?

- [X] I checked I'm using latest Epic Fight and recommended Forge version.

### Is this issue related to mod incompatibility?

- [X] This is a mod compatibility issue and I'm aware of the problem.

### The mod

Mekanism

### Minecraft Version

1.20.1

### What happened?

![2023-10-29_10 23 48](https://github.com/Yesssssman/epicfightmod/assets/126712560/33ef1235-fd5d-4255-9b79-6515fee02662)

**Comment by MetalKnight56** [comment]
Do they use geckolib? If not then this issue is expected.

**Comment by robit4** [comment]
I dosent

**Comment by sorucoder** [comment]
I tried to follow your guide on patching armor models to patch the Mekasuit. I applied the weight paint in such a way that it fits the model's aesthetics. When I tried to export it, Blender had a division by zero error. I patched the exporter to use zero in that case. Regardless, I created the resourcepack, and tried to use it, but no change in-game. I cannot figure out what exactly I am doing wrong. If having this resourcepack can help you in some way, [it can be downloaded here](https://drive.proton.me/urls/7QG7ZPN31W#XPIQHqWfZ2yw).
